src/Microsoft.DotNet.Build.Tasks/Targets/CodeCoverage.targets

```diff
@@ -6,7 +6,7 @@
   -->
   <PropertyGroup>
     <OpenCoverVersion>4.6.166</OpenCoverVersion>
-    <ReportGeneratorVersion>2.3.0.0</ReportGeneratorVersion>
+    <ReportGeneratorVersion>2.3.1</ReportGeneratorVersion>
     <CoverallsUploaderVersion>1.4</CoverallsUploaderVersion>
```

Comment:
Actually i suspect we have the same problem with the CoverallsUploader version. You may need to change this to 1.4.0.